testing/simapp/app.go

```diff
@@ -173,7 +175,9 @@ var (
 		ibcmock.AppModuleBasic{},
 		ica.AppModuleBasic{},
 		authzmodule.AppModuleBasic{},
+		groupmodule.AppModuleBasic{},
 		vesting.AppModuleBasic{},
+		nftmodule.AppModuleBasic{},
```

Comment:
i dont think these are needed for ibc testing?

Reply:
they aren't, unless you want to test ibc with the nft module, and with all of the features of the cosmos-sdk.